skopt/space/space.py

```diff
@@ -214,15 +214,19 @@ def inverse_transform(self, Xt):
     def bounds(self):
         return (self.low, self.high)
 
+    def __contains__(self, point):
+        return self.low <= point < self.high
+
     @property
     def transformed_bounds(self):
+        eps = np.finfo(float).eps
         if self.transform_ == "normalize":
-            return 0.0, 1.0
+            return 0.0, 1.0 - eps
```

Comment:
Is this the best we can do? Or should we modify the upper edge just before the call to BFGS?